### PR TITLE
Use anchor picker in authenticate

### PR DESCRIPTION
**Pull request by nmattia**
This integrates the anchor picker component in the authenticate flow/prompt.

The component supports selecting from multiple anchors, but for simplicity we only support a single anchor for now (to avoid messing with the local storage schema just yet).

### CODE REVIEW

File: src/frontend/src/components/anchorPicker.ts [hunk 1]
```diff
@@ -64,7 +71,7 @@ function elements(props: PickerProps): TemplateResult[] {
         // replace the "use another anchor" button with actual menu when ready (clicked)
         new Promise<void>((resolve) => {
           otherAnchorOpen = resolve;
-        }).then(() => Promise.resolve(otherAnchorMenu({ pick: props.pick }))),
+        }).then(() => Promise.resolve(otherAnchorMenu(props))),
```
Comment (nmattia) [inline src/frontend/src/components/anchorPicker.ts:74]: I know, will try to get rid of this `.then()` later

File: src/frontend/src/components/anchorPicker.ts [hunk 2]
```diff
@@ -115,22 +122,47 @@ function otherAnchorMenu(props: { pick: PickCB }): TemplateResult {
...
-        <button class="c-button c-button--secondary">
-          Create a new anchor
-        </button>
-      </div>
+      ${mkLinks(props)}
     </div>
   `;
 }
+
+const mkLinks = ({
```
Comment (nmattia) [inline src/frontend/src/components/anchorPicker.ts:140]: just moved

**Review by frederikrothenberger — APPROVED** [review]
I like the new box and it works well. 👍

However, I think we should add screenshots for the no anchor selected / "more options" clicked variant of the box.

**Comment by nmattia** [comment]
> However, I think we should add screenshots for the no anchor selected / "more options" clicked variant of the box.

Good call, will do this in a separate PR

**Pull request merged**